ocw_data_parser/ocw_data_parser.py

```diff
@@ -139,9 +145,18 @@ def generate_master_json(self):
                                    for instructor in safe_get(self.jsons[0], "instructors")]
         new_json["language"] = safe_get(self.jsons[0], "language")
         new_json["extra_course_number"] = safe_get(self.jsons[0], "linked_course_number")
-        new_json["course_features"] = safe_get(self.jsons[0], "feature_requirements")
         new_json["course_collections"] = safe_get(self.jsons[0], "category_features")
         new_json["course_pages"] = self.compose_pages()
+        course_features = []
+        feature_requirements = safe_get(self.jsons[0], "feature_requirements")
+        if feature_requirements:
+            for feature_requirement in feature_requirements:
+                for page in new_json["course_pages"]:
+                    if page["short_url"] in feature_requirement["ocw_feature_url"]:
```

Comment:
any chance "ocw_feature_url" might not be in `feature_requirement`?  If so might want to do something like `feature_requirement.get("ocw_feature_url", "")`

Reply:
I'll change these to safe_get as that's probably a good future precaution.  For what it's worth though, I've been testing these changes against all OCW prod content that I downloaded locally and didn't see any errors.